lib/inspec/resources/postgres.rb

```diff
@@ -125,6 +143,23 @@ def version_from_dir(dir)
       end
     end
 
+    def version_from_dir_windows(dir)
+      dirs = inspec.command("Get-ChildItem -Path \"#{dir}\" -Name").stdout
+      entries = dirs.lines.count
+      case entries
+      when 0
+        warn "Could not determine version of installed PostgreSQL by inspecting #{dir}"
```

Comment:
We really should be using Inspec::Log.warn instead of plain warn, here

Reply:
Done.